substrate/frame/balances/src/tests/currency_tests.rs

```diff
@@ -144,7 +144,9 @@ fn lock_removal_should_work() {
 		.monied(true)
 		.build_and_execute_with(|| {
 			Balances::set_lock(ID_1, &1, u64::MAX, WithdrawReasons::all());
+			assert_eq!(System::consumers(&1), 1);
 			Balances::remove_lock(ID_1, &1);
+			assert_eq!(System::consumers(&1), 0);
```

Comment:
Is it possible for the balances pallet to increase consumers more than 1? We could add more tests in that case

Reply:
I don't think it's possible.